src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/PoolBlockPeriodTest.netcoreapp.cs

```diff
@@ -64,126 +63,83 @@ public void TestNonAzureBlockingPeriod(string description, object[] Params)
         [InlineData("Test policy with Auto (lowercase)", new object[] { "auto" })]
         [InlineData("Test policy with Auto (miXedcase)", new object[] { "auTo" })]
         [InlineData("Test policy with Auto (Pascalcase)", new object[] { "Auto" })]
-        public void TestSetPolicyWithAutoVariations(string description, object[] Params)
-        {
-            SqlConnection.ClearAllPools();
-            string policyString = Params[0] as string;
-
-            string connString = CreateConnectionString(_sampleAzureEndpoint, null) + $";{_policyKeyword}={policyString}";
-            PoolBlockingPeriodAzureTest(connString, PoolBlockingPeriod.Auto);
-        }
-
-        [Theory]
         [InlineData("Test policy with Always (lowercase)", new object[] { "alwaysblock" })]
         [InlineData("Test policy with Always (miXedcase)", new object[] { "aLwAysBlock" })]
         [InlineData("Test policy with Always (Pascalcase)", new object[] { "AlwaysBlock" })]
-        public void TestSetPolicyWithAlwaysVariations(string description, object[] Params)
-        {
-            SqlConnection.ClearAllPools();
-            string policyString = Params[0] as string;
-
-            string connString = CreateConnectionString(_sampleAzureEndpoint, null) + $";{_policyKeyword}={policyString}";
-            PoolBlockingPeriodAzureTest(connString, PoolBlockingPeriod.AlwaysBlock);
-        }
-
-        [Theory]
         [InlineData("Test policy with Never (lowercase)", new object[] { "neverblock" })]
         [InlineData("Test policy with Never (miXedcase)", new object[] { "neVeRblock" })]
         [InlineData("Test policy with Never (Pascalcase)", new object[] { "NeverBlock" })]
-        public void TestSetPolicyWithNeverVariations(string description, object[] Params)
+        public void TestSetPolicyWithVariations(string description, object[] Params)
         {
             SqlConnection.ClearAllPools();
             string policyString = Params[0] as string;
-
-            string connString = CreateConnectionString(_sampleNonAzureEndpoint, null) + $";{_policyKeyword}={policyString}";
-            PoolBlockingPeriodNonAzureTest(connString, PoolBlockingPeriod.NeverBlock);
+            PoolBlockingPeriod? policy = null;
+            if (policyString.ToLower().Contains("auto"))
+            {
+                policy = PoolBlockingPeriod.Auto;
+            }
+            else if (policyString.ToLower().Contains("always"))
+            {
+                policy = PoolBlockingPeriod.AlwaysBlock;
+            }
+            else
+            {
+                policy = PoolBlockingPeriod.NeverBlock;
+            }
+            string connString = CreateConnectionString(_sampleAzureEndpoint, null) + $";{_policyKeyword}={policyString}";
```

Comment:
Nit: This could be written as 
`$"{CreateConnectionString(_sampleAzureEndpoint, null)};{_policyKeyword}={policyString}"`